modules/codegen-plugin/src/sbt-test/codegen-plugin/wildcard-config/build.sbt

```diff
@@ -0,0 +1,50 @@
+val scala213 = "2.13.10"
+val scala3 = "3.2.1"
+
+val expectedWildcardArgument = settingKey[String]("The expected value of smithy4sWildcardArgument")
+
+val checkProject = project
+  .in(file("check"))
+  .enablePlugins(Smithy4sCodegenPlugin)
+  .settings(
+    (Compile / expectedWildcardArgument) := "",
+    TaskKey[Unit]("check") := {
+      val got = (Compile / smithy4sWildcardArgument).value
+      val expected = (Compile / expectedWildcardArgument).value
+      val s = streams.value
+      s.log.info(s"testing (version: ${(Compile / scalaVersion).value}) (scalacOptions: ${(Compile / scalacOptions).value})")
+      if (expected != got) {
+        sys.error(s"""expected wildcard argument "$expected", got "$got"""")
+      }
+    },
+  )
+
+val root = project
+  .in(file("."))
+  .enablePlugins(Smithy4sCodegenPlugin)
+  .settings(
+    crossScalaVersions := List(scala213, scala3),
+    scalacOptions ++= {
+      if (scalaVersion.value == scala3) {
+        Seq("-Xfatal-warnings", "-source:future")
+      } else {
+        Seq("-Xfatal-warnings")
+      }
+    },
+    libraryDependencies ++= Seq(
+      "com.disneystreaming.smithy4s" %% "smithy4s-core" % smithy4sVersion.value,
+      "com.disneystreaming.smithy4s" %% "smithy4s-dynamic" % smithy4sVersion.value
```

Comment:
**issue:** I think this is why your test fails. Seems to be unused, so:

```suggestion
```

Reply:
👍 pushed a fix